ingest/anndata_.py

```diff
@@ -61,49 +55,76 @@ def generate_cluster_header(adata, clustering_name):
         else:
             msg = f"Too few dimensions for visualization in obsm \"{clustering_name}\", found {clustering_dimension}, expected 2 or 3."
             raise ValueError(msg)
-        with open(f"{clustering_name}.cluster.anndata_segment.tsv", "w") as f:
+        file_prefix = f"{study_file_id}.{clustering_name}"
+        with open(AnnDataIngestor.set_clustering_filename(file_prefix), "w") as f:
```

Comment:
Nit: airing this out would make it a bit more readable / debuggable:

```suggestion
        filename = AnnDataIngestor.set_clustering_filename(file_prefix)
        with open(filename, "w") as f:
```

Reply:
`86ebd57`